jkube-kit/generator/java-exec/src/test/java/org/eclipse/jkube/generator/javaexec/JavaExecGeneratorTest.java

```diff
@@ -120,4 +124,43 @@ void createAssemblyWithFatJarShouldAddDefaultFileSetsAndFatJar() {
           );
     }
   }
+
+  @Test
+  void customize_whenInvoked_shouldAddLabelsToBuildConfiguration() {
+    // Given
+    properties.put("jkube.generator.java-exec.mainClass", "org.example.Foo");
+    JavaExecGenerator javaExecGenerator = new JavaExecGenerator(generatorContext);
+    List<ImageConfiguration> result = new ArrayList<>();
+
+    // When
+    result = javaExecGenerator.customize(result, false);
+
+    // Then
+    assertThat(result)
+        .singleElement(InstanceOfAssertFactories.type(ImageConfiguration.class))
+        .extracting(ImageConfiguration::getBuildConfiguration)
+        .extracting(BuildConfiguration::getLabels)
+        .asInstanceOf(InstanceOfAssertFactories.MAP)
+        .containsKeys("org.label-schema.build-date", "org.label-schema.description", "org.label-schema.version",
+            "org.label-schema.schema-version", "org.label-schema.build-date", "org.label-schema.name");
+  }
+
+  @Test
+  void customize_whenInvoked_shouldNotAddBuildTimestampToBuildDateLabel() {
+    // Given
+    properties.put("jkube.generator.java-exec.mainClass", "org.example.Foo");
+    JavaExecGenerator javaExecGenerator = new JavaExecGenerator(generatorContext);
+    List<ImageConfiguration> result = new ArrayList<>();
+
+    // When
+    result = javaExecGenerator.customize(result, false);
+
+    // Then
+    assertThat(result)
+        .singleElement(InstanceOfAssertFactories.type(ImageConfiguration.class))
+        .extracting(ImageConfiguration::getBuildConfiguration)
+        .extracting(BuildConfiguration::getLabels)
+        .asInstanceOf(InstanceOfAssertFactories.MAP)
+        .containsEntry("org.label-schema.build-date", LocalDateTime.now().toLocalDate().toString());
```

Comment:
The test should verify that the date is something specific.

Reply:
Do you mean to add a string literal here?

Reply:
Yes, the test should ensure that a date is printed in a given and show it clearly.